byron/ledger/formal-spec/delegation.tex

```diff
@@ -329,131 +333,62 @@ \subsection{Delegation sequences}
   \begin{equation}
     \inference
     {
+      {\begin{array}{l}
+         delegEnv
+      \end{array}}
+      \vdash
+      \trans{\hyperref[eq:sdeleg-bootstrap]{sdeleg}}{}
+      \left(
```

Comment:
I'm not sure we need the parentheses around `delegSt` in this rule

Reply:
thanks, removed ()